data/gegno/gegno I corroboration.txt

```diff
@@ -563,7 +564,7 @@ mission "Return to Giaru Gegno"
 		not "Gegno Intervention: offered"
 		not "Gegno Genocide Defense: offered"
 	on enter
-		dialog `Upon departure from <origin>, you receive an incoming message. "Greetings, human <first> <last>. This message is from Adrauni. Return to <planet> at the first opportunity given to you."`
+		dialog `Upon departure from <origin>, you receive an incoming message on your communications device. "Greetings, human <first> <last>. This message is from Adrauni. Return to <planet> at the first opportunity given to you."`
```

Comment:
This dialog implies that it's still a landing mission, since it references departing from a planet.